### ISSUE TITLE
find.lua error in function verboseCall

### ISSUE THREAD

**Issue by ilichev-andrey** [soumith/cudnn.torch#310]
Hello.

How do I solve the error?

cudnnFindConvolutionForwardAlgorithm failed: 	2	 convDesc=[mode : CUDNN_CROSS_CORRELATION datatype : CUDNN_DATA_FLOAT] hash=-dimA256,3,224,224 -filtA64,3,7,7 256,64,112,112 -padA3,3 -convStrideA2,2 CUDNN_DATA_FLOAT

I would appreciate help!

**Comment by chaneyw** [comment]
I also had this problem. I don't have an exact answer, but it may be some kind of memory issue. I tried running on smaller images, of basically any dimensions, and it works fine at some certain size or lower.

I check my GPU load and the RAM is not full but for some reason beyond a certain size it breaks.

**Comment by ilichev-andrey** [comment]
Noted. An error occurs on the data: 
- 3x96x96
- 3x180x180
- 3x652x121

Although use  https://github.com/facebook/fb.resnet.torch/blob/master/datasets/imagenet.lua#L80 in fb.resnet.torch

Solve the problem adding t.Scale(256) in https://github.com/facebook/fb.resnet.torch/blob/master/datasets/imagenet.lua#L80 ?
```
if self.split == 'train' then
      return t.Compose{
         t.Scale(256),  <--
         t.RandomSizedCrop(224),
         t.ColorJitter({
            brightness = 0.4,
            contrast = 0.4,
            saturation = 0.4,
         }),
         t.Lighting(0.1, pca.eigval, pca.eigvec),
         t.ColorNormalize(meanstd),
         t.HorizontalFlip(0.5),
}
```

**Comment by chaneyw** [comment]
I encountered the error under very different circumstances (evaluating a totally different network than that one), so scaling is not the issue. Thanks though.

**Comment by borisfom** [comment]
Yes there are some cases where cudnnFindConvolutionForwardAlgorithm may fail.
You may try set cudnn.useFindEx. Also make sure you upgrade to  CUDNN 5.1.
